sample-compose/app/src/main/java/co/nimblehq/sample/compose/navigation/Navigation.kt

```diff
@@ -0,0 +1,38 @@
+package co.nimblehq.sample.compose.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavType
+import androidx.navigation.compose.*
+import androidx.navigation.navArgument
+import co.nimblehq.sample.compose.navigation.NavigationRoute.HOME_ROUTE
+import co.nimblehq.sample.compose.navigation.NavigationRoute.SECOND_ARGUMENT_ID
+import co.nimblehq.sample.compose.navigation.NavigationRoute.SECOND_ROUTE
+import co.nimblehq.sample.compose.ui.screens.home.HomeScreen
+import co.nimblehq.sample.compose.ui.screens.second.SecondScreen
+
+object NavigationRoute {
+    const val HOME_ROUTE = "home"
+    const val SECOND_ROUTE = "second"
+
+    const val SECOND_ARGUMENT_ID = "id"
+}
+
+@Composable
+fun Navigation() {
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = HOME_ROUTE) {
+        composable(route = HOME_ROUTE) {
+            HomeScreen(navController = navController)
```

Comment:
Only one argument with the same name seems to be in excess to me. I think we can omit the argument name in this case:
```suggestion
            HomeScreen(navController)
```

Reply:
Resolved: https://github.com/nimblehq/android-templates/discussions/279